src/library/Uno.Themes.WinUI.Markup/Theme/RadioButton.cs

```diff
@@ -0,0 +1,129 @@
+
+						[ResourceKeyDefinition(typeof(Brush), "RadioButtonOuterEllipseCheckedStroke")]
+						public static ResourceValue<Brush> Checked => new("RadioButtonOuterEllipseCheckedStroke", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "RadioButtonOuterEllipseCheckedStrokePointerOver")]
+						public static ResourceValue<Brush> CheckedPointerOver => new("RadioButtonOuterEllipseCheckedStrokePointerOver", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "RadioButtonOuterEllipseCheckedStrokePressed")]
+						public static ResourceValue<Brush> CheckedPressed => new("RadioButtonOuterEllipseCheckedStrokePressed", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "RadioButtonOuterEllipseCheckedStrokeDisabled")]
+						public static ResourceValue<Brush> CheckedDisabled => new("RadioButtonOuterEllipseCheckedStrokeDisabled", true);
+					}
+
+					public static class Fill
+					{
+						[ResourceKeyDefinition(typeof(Brush), "RadioButtonOuterEllipseFill")]
+						public static ResourceValue<Brush> Default => new("RadioButtonOuterEllipseFill", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "RadioButtonOuterEllipseFillPointerOver")]
+						public static ResourceValue<Brush> PointerOver => new("RadioButtonOuterEllipseFillPointerOver", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "RadioButtonOuterEllipseFillPressed")]
+						public static ResourceValue<Brush> Pressed => new("RadioButtonOuterEllipseFillPressed", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "RadioButtonOuterEllipseFillDisabled")]
+						public static ResourceValue<Brush> Disabled => new("RadioButtonOuterEllipseFillDisabled", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "RadioButtonOuterEllipseCheckedFill")]
+						public static ResourceValue<Brush> Checked => new("RadioButtonOuterEllipseCheckedFill", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "RadioButtonOuterEllipseCheckedFillPointerOver")]
+						public static ResourceValue<Brush> CheckedPointerOver => new("RadioButtonOuterEllipseCheckedFillPointerOver", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "RadioButtonOuterEllipseCheckedFillPressed")]
+						public static ResourceValue<Brush> CheckedPressed => new("RadioButtonOuterEllipseCheckedFillPressed", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "RadioButtonOuterEllipseCheckedFillDisabled")]
+						public static ResourceValue<Brush> CheckedDisabled => new("RadioButtonOuterEllipseCheckedFillDisabled", true);
+					}
+
+					public static class HoverRingFill
+					{
+						[ResourceKeyDefinition(typeof(Brush), "MaterialRadioButtonHoverRingFillPointerOver")]
+						public static ResourceValue<Brush> PointerOver => new("MaterialRadioButtonHoverRingFillPointerOver", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "MaterialRadioButtonHoverRingFillPressed")]
+						public static ResourceValue<Brush> Pressed => new("MaterialRadioButtonHoverRingFillPressed", true);
+					}
+
+					public static class Typography
```

Comment:
@agneszitte What about the Typography resources? Are they at the right spot or do they have to be after Theme? (Theme.Typography) Or do we not need them at all since there will be a completely different Theme.Typography?

Reply:
They are at the right spot @rajamatt